front/knowbaseitem.form.php

```diff
@@ -22,4 +22,3 @@
    PluginFormcreatorWizard::footer();
 
 }
-?>
```

Comment:
hi

I checked my branch (up to date): each file ends with at least one empty line. It looks like the diff is misleading :/